src/components/SharedComponents/VideoPlayer/VideoPlayer.tsx

```diff
@@ -173,9 +174,11 @@ const VideoPlayer: FC<VideoPlayerProps> = (props) => {
   return (
     <OakFlex
       $flexDirection={"column"}
-      $width={"100%"}
-      $ba={["border-solid-l"]}
+      $ba={"border-solid-l"}
       $borderColor={"black"}
+      style={{
+        boxSizing: "content-box",
+      }}
```

Comment:
We can rely on the block to fill the horizontal space — setting this to 100% meant that it was impossible to maintain a 16:9 aspect ratio on the container when the child was also sized to 16:9 since the border was rendered inside the box.